### ISSUE TITLE
Converter crashes if number of columns in cols spec exceeds number of columns in table

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-pdf#610]
If the number of columns specified in the cols spec exceeds the number of columns in the table, the converter crashes.

For example:

```
[cols="2*"]
|===
| Foo
|===
```

Here's the error message that gets reported:

```
Table's width was set larger than its contents' maximum width (max width 249.39999999999998, requested 498.79999999999995)
```

Look for similar cases.

**Comment by mojavelinux** [comment]
cc: @ge0ffrey

**Comment by mojavelinux** [comment]
While we fixed a similar problem for #480, we did not consider the case when the number of columns is specified explicitly on an empty (or perceived empty) table.
